bungeecord/src/main/java/com/github/dominik48n/party/bungee/listener/OnlinePlayersListener.java

```diff
@@ -20,24 +20,43 @@
 import com.github.dominik48n.party.api.PartyAPI;
 import com.github.dominik48n.party.user.User;
 import com.github.dominik48n.party.user.UserManager;
+import java.util.logging.Level;
 import net.md_5.bungee.api.connection.ProxiedPlayer;
 import net.md_5.bungee.api.event.PlayerDisconnectEvent;
 import net.md_5.bungee.api.event.PostLoginEvent;
 import net.md_5.bungee.api.plugin.Listener;
+import net.md_5.bungee.api.plugin.Plugin;
 import net.md_5.bungee.event.EventHandler;
 import org.jetbrains.annotations.NotNull;
 
 public class OnlinePlayersListener implements Listener {
 
     private final @NotNull UserManager<ProxiedPlayer> userManager;
+    private final @NotNull Plugin plugin;
 
-    public OnlinePlayersListener(final @NotNull UserManager<ProxiedPlayer> userManager) {
+    public OnlinePlayersListener(final @NotNull UserManager<ProxiedPlayer> userManager, final @NotNull Plugin plugin) {
```

Comment:
Lombok Konstruktor?

Reply:
The reason why this is not implemented can be found in the comment in the pull request.